pr_check.sh

```diff
@@ -29,11 +29,14 @@ function build_image() {
 }
 
 function deploy_ephemeral() {
+    # shellcheck disable=SC2317
```

Comment:
Wouldn't it be easier to disable for the whole script?

Reply:
may be, but I was trying locally and it didn't work putting it on top of the file. Let me try again